homeassistant/components/kmtronic/__init__.py

```diff
@@ -76,7 +76,7 @@ async def async_update_data():
     hass.data[DOMAIN][entry.entry_id] = {
         DATA_HUB: hub,
         DATA_COORDINATOR: coordinator,
-        DATA_REVERSE: _get_config_value(entry, CONF_REVERSE, False),
+        DATA_REVERSE: entry.options.get(CONF_REVERSE, False),
```

Comment:
There's no point in extracting this and save in hass.data. We have access to the config entry in the platform.